docs/model.md

```diff
@@ -1,16 +1,49 @@
 # Model
 
-OTG is an actively developing specification, with contributions coming from real [use cases](/examples/#use-cases). On high level, the model allows to express the following building blocks of a traffic generator configuration:
+## Formal Model
+The formal [model specification](https://github.com/open-traffic-generator/models/blob/master/artifacts/openapi.yaml) can be found on GitHub under [Open Traffic Generator](https://github.com/open-traffic-generator) organization. To explore the model, a viewer friendly [**ReDoc rendering**](https://redocly.github.io/redoc/?url=https://raw.githubusercontent.com/open-traffic-generator/models/master/artifacts/openapi.yaml) is available as well. The OTG APIs support both REST and gRPC interfaces.
+
+## Building Blocks
+
+OTG is an actively developing specification, with contributions from real [use cases](/examples/#use-cases). The model allows to express the following building blocks of a traffic generator configuration:
```

Comment:
Same comment as in the home section PR:
1. "actively developed" instead of "actively developing"?
2. Rephrase last sentence as "The model allows to express configuration of the following building blocks of a traffic generator"?

Reply:
1. done
2. "The model defines the following components of a traffic generator configuration:" is what we settled on